gameobjects/scorearea.cpp

```diff
@@ -11,13 +11,21 @@
 namespace pong
 {
 
-ScoreArea::ScoreArea(float width, float height, bool playerSide)
+ScoreArea::ScoreArea(float width, float height, bool playerSide) :
+    mWidth { width },
+    mHeight { height }
 {
-    mColliderBox = std::make_unique<ColliderBox>(width, height);
     mIsPlayerScoreArea = playerSide;
```

Comment:
mIsPlayerScoreArea = playerSide;

could be moved into the initializer list with the others